src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -293,128 +213,72 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
           <TokenInfoContainer>
             <TokenNameCell>
               <CurrencyLogo currency={currency} size={'32px'} />
-              <Trans>{!token ? 'Name not found' : token.name}</Trans>
-              <TokenSymbol>{token && token.symbol}</TokenSymbol>
+              {tokenName ?? <Trans>Name not found</Trans>}
+              <TokenSymbol>{tokenSymbol ?? <Trans>Symbol not found</Trans>}</TokenSymbol>
               {!warning && <VerifiedIcon size="20px" />}
               {networkBadgebackgroundColor && (
                 <NetworkBadge networkColor={chainInfo?.color} backgroundColor={networkBadgebackgroundColor}>
                   {networkLabel}
                 </NetworkBadge>
               )}
             </TokenNameCell>
+            <TokenActions>
+              {tokenName && tokenSymbol && <ShareButton tokenName={tokenName} tokenSymbol={tokenSymbol} />}
+              <ClickFavorited onClick={toggleFavorite}>
+                <FavoriteIcon isFavorited={isFavorited} />
+              </ClickFavorited>
+            </TokenActions>
           </TokenInfoContainer>
-          <ChartEmpty>
-            <Wave />
-            <Wave />
-          </ChartEmpty>
-          <MissingChartData>
-            <TrendingUp size={12} />
-            Missing chart data
-          </MissingChartData>
+          <ChartContainer>
+            <ParentSize>{({ width, height }) => <PriceChart token={token} width={width} height={height} />}</ParentSize>
+          </ChartContainer>
         </ChartHeader>
-        <MissingData>
-          <AboutSection address={address} tokenDetailData={relevantTokenDetailData} />
-          <StatsSection>
-            <NoInfoAvailable>
-              <Trans>No stats available</Trans>
-            </NoInfoAvailable>
-          </StatsSection>
-          <ContractAddressSection>
-            <Contract>
-              Contract address
-              <ContractAddress>
-                <CopyContractAddress address={address} />
-              </ContractAddress>
-            </Contract>
-          </ContractAddressSection>
-        </MissingData>
+        <AboutSection address={address} tokenDetailData={relevantTokenDetailData} />
+        <StatsSection>
+          <StatPair>
+            <Stat>
+              Market cap
+              <StatPrice>
+                {tokenDetailData.marketCap?.value ? formatDollarAmount(tokenDetailData.marketCap?.value) : '-'}
+              </StatPrice>
+            </Stat>
+            <Stat>
+              24H volume
```

Comment:
does this need to be translated? or are we skipping that?

Reply:
How should something like '24H' be translated? or should I just translate volume?

Reply:
I'm thinkin maybe a sep task to go into tokens/ token details and figure out what/should be <Trans>'d